nautobot/core/jobs/__init__.py

```diff
@@ -50,7 +50,8 @@ def run(self, repository):
             # Given that the above succeeded, tell all workers (including ourself) to call ensure_git_repository()
             app.control.broadcast("refresh_git_repository", repository_pk=repository.pk, head=repository.current_head)
```

Comment:
Is this still necessary with the changes you've made or is it just extra noise? If this fails on one of the workers we don't get any feedback anywhere except the worker console logs right? If so this doesn't function well as a warning that your worker wasn't able to clone the git repo and I can't think of another reason to keep this?

Reply:
Also noting that this causes the worker that ran this job to reload the git repo code a second time.

Reply:
Do we use git repositories to drop arbitrary files on the filesystem of workers? If so, then I think we should keep this.

Reply:
Probably better to keep it for now instead of breaking something. If it becomes an issue we can open another story for it?